.github/CODEOWNERS

```diff
@@ -833,8 +833,8 @@
 /sdk/portal/ci.mgmt.yml @qiaozha @dw511214992 @MaryGao
 
 # PRLabel: %Mgmt
-/sdk/loadtestservice/arm-loadtestservice/ @qiaozha @dw511214992 @MaryGao
-/sdk/loadtestservice/ci.mgmt.yml @qiaozha @dw511214992 @MaryGao
+/sdk/loadtesting/arm-loadtestservice/ @qiaozha @dw511214992 @MaryGao
+/sdk/loadtesting/ci.mgmt.yml @qiaozha @dw511214992 @MaryGao
```

Comment:
Is there any item related with sdk/loadtestservice/loadtesting-rest ?

Reply:
I just found this in the file, didn't find the sdk/loadtestservice/loadtesting-rest